### ISSUE TITLE
Location of critical value of bar scales in code has risk of being overridden in translations

### ISSUE THREAD

**Issue by boisei0** [minvws/nl-covid19-data-dashboard#90]
The critical value, in the code referenced as both “signaalwaarde” and “kritiekeWaarde”, is sourced in the locale. The default value is present in the Dutch locale. If left empty in translations, the Dutch value is used to override the translation. However, if it erroneously isn’t left empty and the value differs from the original, the interpretation strongly changes. 

Should values that act as constants and don’t change upon translating be present in a locale in the first place, rather than be in file with constants? It would also improve the transparency if constants like these are in an easy place to find, rather than having to study the code in order to find them. Furthermore, storing numbers as numerical value rather than as string removes the need for `Number(...)` calls in the code and reduces errors caused by accidental typos slipping into the text values. 

At the moment of writing, the critical value for IC admittances is located at line 102 in the Dutch locale (value 10):
https://github.com/minvws/nl-covid19-data-dashboard/blob/43b7be4c0415607a0f74b78073afe04c5f27ed5a/src/locale/nl.json#L102
and at line 98 in the English locale (value left blank):
https://github.com/minvws/nl-covid19-data-dashboard/blob/43b7be4c0415607a0f74b78073afe04c5f27ed5a/src/locale/en.json#L98

### Governance
- [x] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [X] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

**Comment by VWSCoronaDashboard** [comment]
@boisei0 

Agreed that these numeric values should ideally not live in a locale file!

If you look at an [older version of the app](https://github.com/minvws/nl-covid19-data-dashboard/blob/0.1.1/src/data/textNationaal.json#L159-L185), you can see that at some point not only the text but also all other _configuration_ ended up in a single json files. In hindsight you could say that was premature abstraction of logic.

Not only the _kritieke waarde_ (which has been renamed to _signaal waarde_) was stored there, but also constants like the min and max value and gradients of specific charts.

Long story short, we're planning to remove these constant from configuration files anyway and place them directly in the JSX. 

1. Flatten the configuration file
2. Use it as an actual dictionary (now in develop!)
3. and from that point in time start moving away duplicates, configuration values and other crud from the locale files to better places where possible.

**Comment by VWSCoronaDashboard** [comment]
Configuration values ("signaalwaarde") have been removed from the locale: https://github.com/minvws/nl-covid19-data-dashboard/commit/48ed6cf4efb06cba0c03e0687926716b2f29c356. Thanks for reporting.

**Comment by boisei0** [comment]
This commit introduced a new issue. The “signaalwaarde” is used twice in these components. First in the bar scale, then in the plot below it. 
This line will fail with the changes made, the same with the other components. 
https://github.com/minvws/nl-covid19-data-dashboard/blob/48ed6cf4efb06cba0c03e0687926716b2f29c356/src/components/tiles/IntakeHospital.tsx#L83

**Comment by VWSCoronaDashboard** [comment]
@boisei0 

You are correct!
I've made this PR to resolve that: https://github.com/minvws/nl-covid19-data-dashboard/pull/108
